api/HTMLMediaElement.json

```diff
@@ -2278,7 +2278,8 @@
               {
                 "prefix": "moz",
                 "version_added": "20",
-                "notes": "May be removed in a future release, see <a href='https://bugzil.la/1765201'>bug 1765201</a>."
+                "version_removed": "115",
+                "notes": "Disable by default in version 115 (behind preference <code>dom.media.mozPreservesPitch.enabled</code>). May be fully removed in a future release (see <a href='https://bugzil.la/1765201'>bug 1765201</a>)."
```

Comment:
Normally I'd add the preference as a separate item. However from a user perspective this moz prefix will just disappear, and it is unlikely to be back.
So I think it makes sense to indicate 115 as the last release this was in, and then have the comment about the preference in the notes. If we find the preference has gone, we could just the remove all the notes.

Reply:
Yes, I think this makes total sense this way! 👍